app.py

```diff
@@ -0,0 +1,71 @@
+from flask import Flask, jsonify, request
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
+
+app.config['SQLALCHEMY_DATABASE_URI'] = "postgresql://postgres:farcry1234@localhost/articles"
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+
+ARTICLES = [
+
+]
+
+
+class Articles(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String(200))
+    description = db.Column(db.String(500))
+
+    def __init__(self, title, description):
+        self.title = title
+        self.description = description
+
+
+@app.route("/", methods=["GET"])
+def hello_world():
+    return "<h1>Welcome To the home page we have created rest api and checked it against postman</h1>"
+
+
+@app.route("/add", methods=["POST"])
+def add_article():
+    req_data = request.get_json()
+    title = req_data['title']
+    description = req_data['description']
+

```

Comment:
Add json validator for json verification that the json is valid or not i.e(they contains required keys or not)
Try to use decorator for same